### PR TITLE
[WIP] src/test: add testing for ceph-backport.sh if file is changed

### PR DESCRIPTION
**Pull request by ideepika**
continuation of : https://github.com/ceph/ceph/pull/43027

discussion: https://lists.ceph.io/hyperkitty/list/dev@ceph.io/thread/O7QWWYGWSRP5GQJISQXDLYKIZBEB4PS5/

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by smithfarm** [comment]
I don't feel qualified to approve this PR for merge, and have no idea what impact it will have. I've pinged @batrick and @neha-ojha - hopefully they can help?

File: src/script/test-ceph-backport.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if git $current_pr stats | grep ceph-backport.sh ; then
```
Comment (batrick) [inline src/script/test-ceph-backport.sh:3]: Where does `$current_pr` come from?

File: src/script/test-ceph-backport.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    git clone https://lab.fedeproxy.eu/ceph/ceph-backport
```
Comment (batrick) [inline src/script/test-ceph-backport.sh:4]: This should be something hosted on ceph.com we control.

**Review by batrick — CHANGES_REQUESTED** [review]

File: src/script/test-ceph-backport.sh (hunk 2 above)
Comment (ideepika) [reply src/script/test-ceph-backport.sh:4]: @batrick I don't know much about it, how do we do that?

File: src/script/test-ceph-backport.sh (hunk 1 above)
Comment (ideepika) [reply src/script/test-ceph-backport.sh:3]: it used gitea, which works similar to github API is what I think, but I am not sure, pinged Loic on #ceph-backports

File: src/script/test-ceph-backport.sh (hunk 2 above)
Comment (batrick) [reply src/script/test-ceph-backport.sh:4]: I would ask @djgalloway to mirror it.

**Pull request closed without merging**